core/src/transport.cpp

```diff
@@ -0,0 +1,140 @@
+#include "irods/private/http_api/globals.hpp"
+#include "irods/private/http_api/transport.hpp"
```

Comment:
Change to ...
```cpp
1 | #include "irods/private/http_api/transport.hpp"
2 |
3 | #include "irods/private/http_api/globals.hpp"
```

The header you're implementing should always be listed first. While it's not required, it provides signal to the reader on what's being implemented vs what's acting as a support lib.